__tests__/integrations/php.Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM alpine
+
+ADD packages/php src/packages/php
+
+RUN apk update
+RUN apk add php81 \
+    php81-curl \
+    php81-dom \
+    php81-fileinfo \
+    php81-fpm \
+    php81-mbstring \
+    php81-opcache \
+    php81-openssl \
+    php81-phar \
+    php81-simplexml \
+    php81-tokenizer \
+    php81-xml \
+    php81-xmlwriter
+RUN apk add --update nodejs npm curl
+RUN ln /usr/bin/php81 /usr/bin/php
+
+# Install Composer
+COPY --from=composer /usr/bin/composer /usr/bin/composer
```

Comment:
This pulls down Composer from the `composer` image. I originally tried using the multi-stage workflow like I did in my [HTTPSnippet PHP integration suite](https://github.com/readmeio/httpsnippet/blob/main/integrations/php.Dockerfile) but wasn't able to sort out running composer, from the workflow, twice in two different locations for the PHP SDK deps and again for the demo app deps. This `COPY` flow works great though!